xk6/sm/output.go

```diff
@@ -15,7 +15,10 @@ import (
 	"go.k6.io/k6/output"
 )
 
-const ExtensionName = "sm"
+const (
+	ExtensionName = "sm"
+	RawURLTagName = "__raw_url__"
```

Comment:
I think maybe the double underscores are forbidden in prom? When I run it I get an error that it's an invalid label name. If I change it to a random string it seems to work fine

Reply:
that's weird, I'm replacing that with `url`.

Am I missing some case where the replacement doesn't happen?

Reply:
this is weirder... if I run it "by hand" (which is what I have been doing all along), it's doing the right thing.

Why is it doing something when running from the agent?

Reply:
OK, we figured this out. It's working as expected.